hapi-utils/src/main/java/com/hederahashgraph/fee/FeeBuilder.java

```diff
@@ -193,15 +191,11 @@ public static int getAccountKeyStorageSize(Key key) {
     if (key == Key.getDefaultInstance()) {
       return 0;
     }
-    int keyStorageSize = 0;
-    try {
-      int[] countKeyMetatData = {0, 0};
-      countKeyMetatData = calculateKeysMetadata(key, countKeyMetatData);
-      keyStorageSize = countKeyMetatData[0] * KEY_SIZE + countKeyMetatData[1] * INT_SIZE;
```

Comment:
This code snippet does not throw any more or now we let the exception escalate up the stack trace?

Reply:
I might be wrong but I don't see how `calculateKeysMetadata` would throw an exception.